<?xml version="1.0"?>
<sdf version="1.6">
  <model name="pothole_patch">
    <static>true</static>
    <link name="pothole_link">
      <pose>0 0 0 0 0 0</pose>

      <visual name="pothole_visual">
        <geometry>
          <cylinder>
            <radius>0.5</radius>
            <length>0.001</length>
          </cylinder>
        </geometry>

<scale>1 1 1</scale>


        <!-- This scale is irrelevant now; you can keep or remove -->
        <scale>1 1 1</scale>

        <material>
          <script>
            <uri>file://media/materials/scripts/gazebo.material</uri>
            <name>Gazebo/Black</name>
          </script>
        </material>
      </visual>

      <!-- no collision so it remains non-obstructive -->
    </link>
  </model>
</sdf>
